autotest/gdrivers/esric.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env pytest
+###############################################################################
+#
+# Project:  GDAL/OGR Test Suite
+# Purpose:  Test read functionality for Esri Compact Cache driver.
+# Author:   Lucian Plesea <lplesea at esri.com>
+#
+###############################################################################
+# Copyright (c) 2020, Esri
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+###############################################################################
+
+# from osgeo import osr
+
+from osgeo import gdal
+import gdaltest
+import pytest
+
+
+###############################################################################
+# Verify we have the driver.
+
+def test_esric_1():
```

Comment:
you can avoid this first test, and the test for esric_drv in test_esric_2, and replace it by just
```
pytestmark = pytest.mark.require_driver('ESRIC')
```

Reply:
OK, now it fails because pytestmark isn't used.  Is calling the function sufficient?